### ISSUE TITLE
frecency threw errors when nvim first starts up

### ISSUE THREAD

**Issue by csessh** [nvim-telescope/telescope-frecency.nvim#243]
I recently ran Lazy update and frecency is updated from 39f70a8 to a7ebbb4. 

nvim starts to throw errors at startup:
![Screenshot from 2024-08-18 18-47-21](https://github.com/user-attachments/assets/661932a7-f717-4c36-a9c7-3caff19ad40d)

This happens when: 

if I open nvim without specifying a file, the first time I open up Telescope frecency it will raise exception as shown above. sub-sequence execution will not fail. 

if I open nvim specifying a file, nvim immediately prompt this exception and sub-sequence execution will not fail. 

After checking out `lazy-lock.json` and Lazy restore, the problem goes away. 

![Screenshot from 2024-08-18 18-52-46](https://github.com/user-attachments/assets/5962e646-a7c9-4272-82f7-e6f5637ffc97)

Here's my very simple config for frecency, if it's of any help:
```lua
return {
    "nvim-telescope/telescope-frecency.nvim",
    dependencies = {
        "nvim-tree/nvim-web-devicons",
        "nvim-telescope/telescope.nvim",
    },
    config = function()
        require("telescope").load_extension "frecency"
        vim.keymap.set("n", "<leader>ff", ":Telescope frecency<CR>", {})
    end,
}
```

nvim version: v0.10.1

**Comment by delphinus** [comment]
Reproduced. Can you test with `branch = "fix/joining-errors"? (#243) If solved, I will merge it.

**Comment by delphinus** [comment]
hmm, I think I have solved another problem, maybe. But please try again with the new master (ee2af42) @csessh

**Comment by csessh** [comment]
Can confirm https://github.com/nvim-telescope/telescope-frecency.nvim/commit/ee2af42d5f13d72d877c1c4bcb97918f7428f694 fixes the issue for me. 

Thanks for the swift response, @delphinus.

**Comment by subsid** [comment]
Hey @delphinus fwiw, I'm still seing this error on startup with [9d4d8d2](https://github.com/nvim-telescope/telescope-frecency.nvim/commit/9d4d8d21f629d1b439cfd1b6baaa67c91cccb32f)

![2024-08-20-074133_1902x836_scrot](https://github.com/user-attachments/assets/9239e33c-c5ae-4213-bbb3-c386a5b9efd3)

![2024-08-20-074325_1072x190_scrot](https://github.com/user-attachments/assets/8cc02b9d-bee9-4c6e-8711-cb180f60f731)

**Comment by delphinus** [comment]
@subsid I write a short fix for this: #246. Does this solve your problem?

**Comment by subsid** [comment]
@delphinus Yep, that did it! :partying_face:

**Comment by delphinus** [comment]
Thank you @subsid. I changed code a bit, and merged: 673585e . If you see errors again, feel free to reopen here.
